air_example.toml

```diff
@@ -21,7 +21,7 @@ include_dir = []
 # Exclude files.
 exclude_file = []
 # Exclude specific regular expressions.
-exclude_regex = ["_test.go"]
+exclude_regex = ["_test\\.go"]
```

Comment:
`_test\\.go`  should be `_test\.go`

Reply:
Without `` the string won't escape the \ so you need to escape it once for the "" string, and then again to escape the . So \\ I think.

Reply:
Sorry I didn't realize github didn't escape, I mean't I think it should be \\\\